.travis.yml

```diff
@@ -55,7 +55,21 @@ matrix:
     - jdk: openjdk8
       env:
         - DESC="spotbugs,pmd"
-        - CMD="export MAVEN_OPTS='-Xmx2000m' && mvn -e clean compile pmd:check spotbugs:check"
+        - CMD="export MAVEN_OPTS='-Xmx2000m' && mvn -e clean compile pmd:check spotbugs:check -Dspotbugs.skip=false"
+        - USE_MAVEN_REPO="true"
+
+    # spotbugs and pmd (openjdk11)
+    - jdk: openjdk11
+      env:
+        - DESC="spotbugs,pmd"
+        - CMD="export MAVEN_OPTS='-Xmx2000m' && mvn -e clean compile pmd:check spotbugs:check -Dspotbugs.skip=false"
```

Comment:
please explain why `-Dspotbugs.skip=false` i required.

Reply:
It seems to be unnecessary if the target is specified explicitly, thanks.